Lib/test/test_asyncio/test_subprocess.py

```diff
@@ -1,5 +1,7 @@
+import os
```

Comment:
Perhaps `os` and `time` modules are not needed anymore

Reply:
Good catch!